src/Classes/SimpleFieldSim3D.m

```diff
@@ -38,10 +45,31 @@
         end
         
         function str = toString(this)
+        % Function that generates a string
+        %
+        % Parameters:
+        %
+        %    this:      Self-Reference
+        %
+        % Returns:
+        %
+        %    str:       Returned String
+
             str = this.string;
         end
         
         function graphicsHandle = initPlot3D(this, ax)
+        % Function that initiates a 3D plot
+        %
+        % Parameters:
+        %
+        %    this:              Self-Reference
+        %    ax:                
+        %
+        % Returns:
+        %
+        %    graphicsHandle:    
```

Comment:
@adhusch , Could you please provide a description of the graphicsHandle? Also could you check if the existing descriptions are correct?

Reply:
Alright, thanks, i'll keep this in mind for future PRs